src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -86,11 +86,26 @@
       [:div {:style {:padding "1em 0 0 1em" :fontWeight 600}}
        (get-in props [:data :name])]])})
 
+(defn- get-workspace-name-string [ws]
+  (str (get-in ws ["workspace" "namespace"]) "/" (get-in ws ["workspace" "name"])))
+
+(defn- get-max-workspace-name-length [workspaces]
+  (apply max (map count (map #(get-workspace-name-string %) workspaces))))
+
+(defn- get-workspace-description [ws]
+  (get-in ws ["workspace" "attributes" "description"]))
```

Comment:
I've been thinking about that exact thing too - I think I have a way to do that.